modules/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher/src/Microsoft.Azure.IIoT.Modules.OpcUa.Publisher.csproj

```diff
@@ -34,5 +39,8 @@
     <None Update="appSettings.Development.json">
       <CopyToOutputDirectory>Always</CopyToOutputDirectory>
     </None>
+    <None Update="publishednodes.json">
```

Comment:
Will publisher now always start in standalone?

Reply:
No, but if the file is there (for development), it will be copied to the output. But this **only** affects development.